pandas/plotting/_core.py

```diff
@@ -334,7 +334,11 @@ def result(self):
     def _compute_plot_data(self):
         data = self.data
 
-        if isinstance(data, Series):
+        from pandas import Series
```

Comment:
The first commit for this PR led to a bunch of `TypeError`s on Travis that I can't reproduce locally.  My best/only guess as to why is that the check `isinstance(data, ABSeries)` (now at L341) is somehow evaluating differently than the original `isinstance(data, Series)` (previously at L337).

So 337-339 is an attempt to check this guess.  Regardless of whether the guess is correct, this check will be removed before long.